src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -104,10 +104,10 @@
 
     /**
      * The "global" id of this conference, set by the controller (e.g. jicofo)
-     * as opposed to the bridge. This defaults to {@code null} unless it is
-     * specified.
+     * as opposed to the bridge. This is a 32-bit unsigned integer, or {@code -1}
```

Comment:
Could we do a constant for the `-1` value, since we use it for health checks?

Reply:
I removed the use for health checks. It's used in Videobridge and VideobridgeShim.